### DISCUSSION TITLE
ValueError: SPI(2) doesn't exist after build ESP32 with IDF_5.0.2

### DISCUSSION THREAD

**Discussion by styliter** [community/community#59876] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I build eps32 GENERIC with IDF_5.0.2 and get firmware.bin successfully.

There is an error on SPI(2):

>>> from machine import SPI
>>> spi = SPI(2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: SPI(2) doesn't exist

>>> SPI(1)
SPI(id=1, baudrate=500000, polarity=0, phase=0, bits=8, firstbit=0, sck=14, mosi=13, miso=12)

**Comment by whatdtech** [comment]
same issue here. Did you find any solution?
